CMakeLists.txt

```diff
@@ -296,6 +296,10 @@ include_directories( build_extra )
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+if (CMAKE_VERSION VERSION_LESS 3.1)
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --std=c++${CMAKE_CXX_STANDARD}") # [[version]] needed for Debian 8
+endif()
+
```

Comment:
1. CMAKE_CXX_STANDARD was introduced in [cmake 3.1](https://cmake.org/cmake/help/v3.1/release/3.1.0.html). 
2. There's not --std=c++14 flag on [MSVC](https://blogs.msdn.microsoft.com/vcblog/2016/06/07/standards-version-switches-in-the-compiler/)
3. Cmake 3.1 was released in [December 2014](https://cmake.org/pipermail/cmake/2014-December/059418.html) like [c++14](https://en.wikipedia.org/wiki/C%2B%2B14). If we support c++14 we should support cmake 3.1 too.